src/MCPClient/lib/clientScripts/elasticSearchIndexProcessAIP.py

```diff
@@ -21,9 +21,9 @@
 # @subpackage archivematicaClientScript
 # @author Mike Cantelon <mike@artefactual.com>
 import sys, os, time, ConfigParser
-os.environ['DJANGO_SETTINGS_MODULE'] = 'settings.common'
-sys.path.append("/usr/lib/archivematica/archivematicaCommon")
-sys.path.append("/usr/lib/archivematica/archivematicaCommon/externals")
+
+# archivematicaCommon
+# externals
```

Comment:
Ignore that, I forgot about `xmltodict`. Probably don't need to comment it here though?